v2/src/log_sink_etw.c

```diff
@@ -515,6 +519,12 @@ static void internal_emit_self_described_event_va(const char* event_name, uint16
                             /* Codes_SRS_LOG_SINK_ETW_01_080: [ If the property type is LOG_CONTEXT_PROPERTY_TYPE_uint8_t, the event data descriptor shall be filled with the value of the property by calling EventDataDescCreate. ]*/
                             EventDataDescCreate(&_tlgData[_tlgIdx], context_property_value_pairs[i].value, sizeof(uint8_t));
                             break;
+                        case LOG_CONTEXT_PROPERTY_TYPE_bool:
+                            int32_t* context_property_value_32_bits = malloc(sizeof(int32_t));
+                            context_property_value_32_bits = (int32_t*)context_property_value_pairs[i].value;
```

Comment:
Uhm, we cannot malloc here, we should simply have a stack variable and give the address of that.

Reply:
Besides,  this would leak, right? How would it pass the gate?